### PR TITLE
Update colors, cssnano and more-css

### PR DESCRIPTION
**Pull request by shinnn**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,225 +13,225 @@
...
 <th><a href="https://github.com/css/csso.git">csso - 1.3.11</a></th>
 <th><a href="https://github.com/css/csso.git">csso (reordering off) - 1.3.11</a></th>
 <th><a href="https://github.com/stoyan/cssshrink.git">cssshrink - 0.0.5</a></th>
 <th><a href="https://github.com/hail2u/node-csswring.git">csswring - 3.0.4</a></th>
-<th><a href="https://github.com/army8735/more.git">more-css - 0.10.3</a></th>
+<th><a href="https://github.com/army8735/more.git">more-css - 0.11.0</a></th>
 <th><a href="https://github.com/kurakin/ncss.git">ncss - 1.1.1</a></th>
 <th>sqwish - 0.2.2</th>
-<th><a href="http://github.com/yui/ycssmin.git">ycssmin - 1.0.1</a></th>
+<th><a href="ssh://git@github.com/yui/ycssmin.git">ycssmin - 1.0.1</a></th>
```
Comment (XhmikosR) [inline index.html]: Use http.

Comment (shinnn) [inline index.html]: As I said here https://github.com/GoalSmashers/css-minification-benchmark/pull/37#discussion_r29122009, the build script always produces invalid URLs.

I'll fix it by hand for now.

Comment (XhmikosR) [inline index.html]: Well, you only have this issue because you use SSH.

**Comment by shinnn** [comment]
Fixed.

@XhmikosR Thanks for reviewing.

**Pull request merged**